go.mod

```diff
@@ -20,6 +20,7 @@ require (
 	github.com/klauspost/compress v1.16.7
 	github.com/klauspost/pgzip v1.2.6
 	github.com/manifoldco/promptui v0.9.0
+	github.com/mattn/go-sqlite3 v1.14.17
```

Comment:
@mtrmac I think it would have been better to consider a CGo-free module like https://pkg.go.dev/modernc.org/sqlite. More info https://gitlab.com/cznic/sqlite